backend/private-graph/graph/schema.resolvers.go

```diff
@@ -3070,7 +3070,7 @@ func (r *mutationResolver) UpdateLogAlert(ctx context.Context, id int, input mod
 
 	if err := r.DB.Model(&model.LogAlert{Model: model.Model{ID: id}}).
 		Where("project_id = ?", input.ProjectID).
```

Comment:
is the problem that we should be searching by alert id? `Where("id = ", input.ID)`, that way GORM will know that this is an existing entry

Reply:
Doesn't `{Model: model.Model{ID: id}})` already match the alert id? That's how the other alerts are being updated and it works for those.

Reply:
oh interesting, TIL